.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Docker Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release_cass_operator:
+    name: Release Cass Operator
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login Skopeo DockerHub
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | skopeo login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin docker.io
+      - name: Login Skopeo ECR
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.ECR_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.ECR_SECRET }}
+        run: aws ecr get-login-password --region us-east-1 | skopeo login -u AWS --password-stdin ${ECR_REPO}
+      - name: Publish to Dockerhub
+        env:
+          ECR_REPO: ${{ secrets.ECR_REPO }}
+        run: |
+          GITHUB_REPO_URL="https://github.com/${{ github.repository }}"
+          GIT_TAG="${GITHUB_REF##*/}"
+          VERSION=${GIT_TAG#v} # strip the initial 'v' from the tag to get the version
+
+          DOCKERHUB_REPOSITORY="datastax/cass-operator"
+          ECR_REPOSITORY="${ECR_REPO}/datastax/cass-operator"
```

Comment:
We are going to reference ECR instead of github packages as github packages does not support multiarch images.